### PR TITLE
Update Planet dashboard image in Admin User Manual

### LINKED ISSUE #2693: Update Planet dashboard image in Admin User Manual
### Problem
Outdated Planet dashboard images

### Steps to reproduce the problem
N/A

### Screenshots

Screen Shot 2019-08-29 at 1 28 22 PM
Screen Shot 2019-08-29 at 1 28 55 PM
Screen Shot 2019-08-29 at 1 32 35 PM

Planet dashboard images need to change
Language Overview

Screen Shot 2019-08-29 at 2 44 53 PM

Calendar Overview and Email Overview

There are no Calendar and Email in the latest version of Planet

Screen Shot 2019-08-29 at 2 49 15 PM

### Proposed solution
Update Planet dashboard images

### PR DESCRIPTION
**Pull request by pattanawadee88**
Fixes #2693

### Description
Outdated Planet dashboard images

### Raw.Githack preview link
https://raw.githack.com/pattanawadee88/pattanawadee88.github.io/pat-admin-branch/#!./pages/techgenius/tg-planet-user-manual.md

[x] Check for issue number in pull request title
[x] Are there any unneeded files in the pull request?
[x] Did they make a branch for their patch?
[x] Does the pull request actually fix the issue?
[x] Check the pull request on raw.githack, does it display without any errors?
[x] Is there any merge conflicts?
[x] Make sure that people use their GitHub accounts when making commits through git

### CODE REVIEW

File: pages/techgenius/tg-planet-user-manual.md [hunk 1]
```diff
@@ -497,10 +460,17 @@ In order to update software and receive Publications,
...
-  4. The update will automatically run and may take several minutes depending on your internet connection. While the loading screen is still spinning, remain connected and wait until you get a message notifying you that the software update has been successful.
+  2. Checking your software by selecting the manager setting from the top right corner.
+  
+  ![Software check](images/tg-software-update.png)
+  
+  3. As shown below, showing software of Nation Version and Local Version.
+  
+  ![Software version](images/tg-software-update3.png)
+  
+  4. **Upgrade planet apk** will allow you to update the software and sync Publications.
```
Comment (lmmrssa) [inline pages/techgenius/tg-planet-user-manual.md]: upgrade planet apk is to get latest myplanet not to update software. Updating software is done by **Update**
Also there is no publication sync with this. It is just upgrading.
There is **Manage Sync** to sync data. NOTE: No publications concept in Planet anymore

File: pages/techgenius/tg-planet-user-manual.md [hunk 2]
```diff
@@ -497,10 +460,17 @@ In order to update software and receive Publications,
...
+  
+  ![Software update](images/tg-software-update2.png)
+   
+
+  5. The update will run and may take several minutes depending on your internet connection. While the loading screen is still spinning, remain connected and wait until you get a message notifying you that the software update has been successful.
```
Comment (lmmrssa) [inline pages/techgenius/tg-planet-user-manual.md]: There is verify button shown once update is done successfully. So you can verify if upgrade was successful or had some error

File: pages/techgenius/tg-planet-user-manual.md [hunk 3]
```diff
@@ -447,45 +447,8 @@ Learners can give feedback for any page or Resource at any time by clicking on 
 
 Learners can change the language of the Planet system at any time by going to the Home page and selecting the language pull down menu from in the top right corner.
```
Comment (lmmrssa) [inline pages/techgenius/tg-planet-user-manual.md]: Menu bar is always available so no need to be on HomePage.

File: pages/techgenius/tg-planet-user-manual.md [hunk 4]
```diff
@@ -497,10 +460,17 @@ In order to update software and receive Publications,
...
-   ![Dashboard Software Update](images/tg-software-overview-update.png)
-
-  3. These two gray buttons in red writing will allow you to update the software and sync Publications. First, click “Update Available (0.12.33)”.
-
-  4. The update will automatically run and may take several minutes depending on your internet connection. While the loading screen is still spinning, remain connected and wait until you get a message notifying you that the software update has been successful.
+  2. Checking your software by selecting the manager setting from the top right corner.
+  
+  ![Software check](images/tg-software-update.png)
+  
+  3. As shown below, showing software of Nation Version and Local Version.
```
Comment (lmmrssa) [inline pages/techgenius/tg-planet-user-manual.md]: You will only get update if version do not match.

File: pages/techgenius/tg-planet-user-manual.md (hunk 2 above)
Comment (lmmrssa) [inline pages/techgenius/tg-planet-user-manual.md]: Not indented properly. So numbering starts from **1** again

**Comment by lmmrssa** [comment]
@pattanawadee88 there is still issue with indentation of list. Please check the raw.githack link to make sure it looks good

**Comment by pattanawadee88** [comment]
> Looks good. Wondering are you familiar with using git to make changes locally? That is something you will have to look through. As that will be the use case in later phases.

I am not familiar with using git to make changes locally but I could study for that.

**Pull request merged**